src/components/Context/PageTransitionContext.js

```diff
@@ -0,0 +1,29 @@
+import { createContext, useEffect, useState } from 'react';
```

Comment:
This file really acting like its imported somewhere. You should get deleted 🌩️🌩️⚡⚡⚡ NOWW. MODS, CRUMBLE HIS COMMIT HISTORY!!